app/assets/stylesheets/bootstrap_and_overrides.scss

```diff
@@ -0,0 +1,193 @@
+
+.header h1 a  {
+  text-decoration: none;
+  color: black;
+}
+
+a:hover {
+  text-decoration: none;
+}
+
+.header {
+  h1, .h1 {
+      font-size: 3em;
+      font-weight: 400;
+  }
+
+  h2, .h2 {
+      font-size: 1.15em;
+      font-weight: 200;
+      margin-top: 0;
+  }
+}
+
+.container-fluid.black {
+  margin-left: -15px;
+  margin-right: -15px;
+  background: black;
+}
+
+.nav-pills > li > a {
+  border-radius: 0;
+  color: whitesmoke;
+
+}
+
+.nav-pills > li > a:hover {
+  color: $taylor-red;
+
+}
+
+#content-padding {
+  height: 11em;
+  //height: 200px;
+}
+
+.post {
+  position: relative;
+
+}
+
+.panel-footer {
+  bottom: 0;
+  position: absolute;
+  width: 100%;
+
+  h6 {
+    color: #B3B3B3;
+  }
+}
+
+.panel.post-showcase {
+    position: relative;
+    height: 17em;
+    border: 2px solid #E4E4E4;
+    border-radius: 0;
+    box-shadow: 0 0 0.25em #D2D2D2;
+    margin-bottom: 20px;
+
+    h5 {
+        padding-bottom: 0.25em;
+        border-bottom: 1px solid lighten($taylor-red, 50%);;
+        transition: text-shadow 0.5s;
+
+        a {
+          font-size: 1.5em;
+          color: $taylor-red;
+        }
+    }
+    h5:hover {
+      text-decoration: none;
+      text-shadow: 0 2px 1.0425em #C3C2C2;
+    }
+}
+
+
+#navbar {
+  background: white;
+}
+
+.next-arrow {
+    position: fixed;
+    bottom: 60px;
+    text-align: center;
+    font-size: 5em;
+    max-height: 0.6em;
+    width: 100%;
+
+    .arrow-container {
+      width: 1.5em;
+      height: 1.1em;
+      display: inline-block;
+      border-radius: 22px;
+      background: rgba(1,1,1,0.2);
+    }
+}
+
+
+
+.next-arrow span {
+    transition: font-size 0.25s, text-shadow 0.25s;
+}
+
+.next-arrow span:hover {
+  font-size: 105%;
+  text-shadow: 0 0 0.0625em #A5A5A5;
+}
+
+.alert {
+  border: none;
+  border-radius: 0;
+  box-shadow: 0 0 5px #C5C5C5;
+}
+
+.alert-info {
+  background-color: #2B82D0;
+  color: #FFFFFF;
+}
+
+.alert-danger {
+  background-color: #D02B2B;
+  color: #FFFFFF;
+}
+
+.form-mobile input.btn.btn-primary {
+  width: 100%;
+}
+
+/* About */
+.photo img {
+  width: 9em;
+  margin-right: 1em;
+}
+
+
+/* Extra small devices (phones, less than 768px) */
+/* No media query since this is the default in Bootstrap */
+
+/* Small devices (tablets, 768px and up) */
+@media (min-width: $screen-sm-min) {
+  .comments-content {
+    margin-left: 0;
+    margin-right: 0;
+  }
+
+  textarea.form-control.comment-input {
+    height: 8em;
+  }
+
+  .form-mobile input.btn.btn-primary {
+    width: auto;
+  }
+
+  /* About */
+  // #about-content {
+  //   display: block;
+  //   margin-left: auto;
+  //   margin-right: auto;
+  // }
+
+  .photo img {
+    width: 14em;
+    margin-right: 1em;
+  }
+
+
+}
+
+/* Medium devices (desktops, 992px and up) */
+@media (min-width: $screen-md-min) {
+  .panel.post-showcase h5 a {
+    font-size: 1.75em;
+  }
+
+}
+
+/* Large devices (large desktops, 1200px and up) */
+@media (min-width: $screen-lg-min) {
+
+}
```

Comment:
Small visual detail. You can add a very specific selector so that your dropdown menu icon stays illuminated while the menu is active. 

```.nav .open > a, .nav .open > a:hover, .nav .open > a:focus {
    color: #D00000;
}```